test/integration/scheduler_perf/dra/dra_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
```

Comment:
Can you add an OWNERS file with:

```
# See the OWNERS docs at https://go.k8s.io/owners

reviewers:
  - bart0sh
  - klueska
  - pohly
labels:
  - wg/device-management
```

The label helps us track pending PRs (see also https://github.com/kubernetes/kubernetes/pull/128379).